### PR TITLE
Feat: Secondary Token Balances

### PR DESCRIPTION
**Pull request by Adamj1232**
### Description

This PR adds the ability to pass in an array of  `secondaryTokens` per chain object within the W3O init and have those balances pulled from the associated chain RPC and be available within the wallet accounts object to the DAPP and displayed within Account Center.

```typescript

interface Chain {
 ....
  /**
   * An optional array of tokens to be available to the dapp in the
   * app state object per wallet within the wallet account and displayed
   * in Account Center (if enabled)
   */
  secondaryTokens?: SecondaryTokens[]
}

interface SecondaryTokens {
  /**
   * Required - The onchain address of the token associated
   * with the chain it is entered under
   */
  address: string
  /**
   * Required - The symbol of the token i.e. USDC, ETH, 1INCH
   */
  name: TokenSymbol
  /**
   * An optional svg or url string for the icon of the token.
   * If an svg is used ensure the height/width is set to 100%
   */
  icon?: string
}
```

This PR also includes a refactor/styling of Account Center to match the latest designs and handle display depending on the appMetaData, secondary tokens, etc passed in on init

account-center-example
Screenshot 2023-05-03 at 4 14 31 PM
Screenshot 2023-05-03 at 4 04 06 PM
Screenshot 2023-05-03 at 4 03 20 PM

### Checklist
- [x] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [x] Check the box that allows repo maintainers to update this PR
- [x] Test locally to make sure this feature/fix works
- [x] Run `yarn check-all` to confirm there are not any associated errors
- [x] Confirm this PR passes Circle CI checks
- [x] Add or update relevant information in the documentation

### CODE REVIEW

**Review by aaronbarnardsound — APPROVED** [review]
Looks good. One question, if an icon is not passed in for a token does it just render a blank space? Maybe we should have a default token icon?

**Comment by Adamj1232** [comment]
> Looks good. One question, if an icon is not passed in for a token does it just render a blank space? Maybe we should have a default token icon?

Currently we just render a blank space per the designs but I had originally had a question mark icon which could be a little confusing for the end user. Any other ideas @gesquinca?

File: packages/core/src/views/connect/Index.svelte [hunk 1]
```diff
@@ -339,6 +340,19 @@
         })
       })
     }
+    if (
+      !secondaryTokens &&
+      Array.isArray(appChain.secondaryTokens) &&
```
Comment (pavelivanov) [inline packages/core/src/views/connect/Index.svelte:345]: it's possible that appChain will be "undefined" if the chain selected in user's wallet doesn't exist in the config. This leads to an error:

> Cannot read properties of undefined (reading 'secondaryTokens')

:(

**Comment by Adamj1232** [comment]
@pavelivanov we have a patch coming out for this shortly - apologies for any inconvenience

File: packages/core/src/views/connect/Index.svelte (hunk 1 above)
Comment (Adamj1232) [reply packages/core/src/views/connect/Index.svelte:345]: @pavelivanov we have a patch coming out for this shortly - apologies for any inconvenience

Comment (kanzure) [reply packages/core/src/views/connect/Index.svelte:345]: > we have a patch coming out for this shortly - apologies for any inconvenience

is this patch available?

Comment (Adamj1232) [reply packages/core/src/views/connect/Index.svelte:345]: There is a fix for this on @web3-onboard/core v2.19.0-alpha.1 and an official release will be out shortly.

**Pull request merged**